### PR TITLE
Auto-Complete Required Request Body Parameters when Inserting Commands

### PR DESCRIPTION
**Pull request by Snehil-Shah**
Instead of returning an empty request body when inserting commands from the side panel (like it does now), we can return the required fields in the request body. This will make the commands panel truer to its use, ie. making it easier for the developer to find and write requests.

they can just insert it from the commands panel and quickly fill in the request body values, and if they want to customize it more, the [DOCS button](https://github.com/qdrant/qdrant-web-ui/pull/142) will always be there!

- **Preview**: (wait for the GIF to load)

params

**Exceptions**: This won't auto-complete request bodies that can have multiple options for a single parameter [as shown below](https://qdrant.github.io/qdrant/redoc/index.html#tag/cluster/operation/update_collection_cluster) (wait for the GIF to load)

Multiple

This should be the preferred way as well. Inserting any one of the possible parameters might limit the user to just one operation, so it's better to encourage them to refer to the documentation.

### CODE REVIEW

File: src/components/CodeEditorWindow/Menu/CommandsDrawer/CommandsDrawer.jsx [hunk 1]
```diff
@@ -8,6 +8,19 @@ import useMediaQuery from '@mui/material/useMediaQuery';
...
+    const refPath = requestBodyContent[contentType].schema.$ref;
+    // parse and navigate to the ref
+    const pathComponents = refPath.slice(2).split('/');
+    const schema = pathComponents.reduce((doc, pathComponent) => doc[pathComponent], openapi);
+    return schema.required;
+  } else {
+    return requestBodyContent[contentType].schema.required;
+  }
+};
+
```
Comment (Snehil-Shah) [inline src/components/CodeEditorWindow/Menu/CommandsDrawer/CommandsDrawer.jsx:23]: @kartik-gupta-ij Should I move this function to some other utils file?

Comment (kartik-gupta-ij) [reply src/components/CodeEditorWindow/Menu/CommandsDrawer/CommandsDrawer.jsx:23]: yeah sure!

Comment (Snehil-Shah) [reply src/components/CodeEditorWindow/Menu/CommandsDrawer/CommandsDrawer.jsx]: @kartik-gupta-ij I created a `CommandsDrawerUtils` file and put this function there. is that okay?

Comment (kartik-gupta-ij) [reply src/components/CodeEditorWindow/Menu/CommandsDrawer/CommandsDrawer.jsx]: Seems good to me!

**Comment by generall** [comment]
Thanks @Snehil-Shah, we will include this changes into the next release!

**Review by generall — APPROVED** [review]

**Pull request merged**